src/pages/Game.js

```diff
@@ -52,6 +62,17 @@ export const Game = () => {
         )}
       </div>
       <div className='infoDisplay'>{info && <Info data={info} />}</div>
+      <div className='button'>
+        {isFinished && (
+          <Button
+            onClick={() => {
+              if (isFinished) navigate('/')
```

Comment:
Since we render the button only if the game is finished, do we need to perform the `isFinished()` check before navigating?